clients/admin-ui/src/features/taxonomy/hooks.tsx

```diff
@@ -78,6 +78,7 @@ const transformTaxonomyBaseToInitialValues = (
   description: t.description ?? "",
   parent_key: t.parent_key ?? "",
   is_default: t.is_default ?? false,
+  version_added: t.version_added ?? "",
```

Comment:
Looking more closely at the db here, I think we should need to write back the values of `version_deprecated` and `replaced_by`, or we could end up overwriting those if they are populated.